packages/suite/src/actions/wallet/coinjoinAccountActions.ts

```diff
@@ -92,11 +92,12 @@ const coinjoinAccountPreloading = (isPreloading: boolean) =>
         },
     } as const);
 
-const coinjoinSessionPause = (accountKey: string) =>
+const coinjoinSessionPause = (accountKey: string, interrupted: boolean) =>
     ({
         type: COINJOIN.SESSION_PAUSE,
         payload: {
             accountKey,
+            interrupted,
```

Comment:
In this context `interrupted` means it was force paused, so it is paused but for other reason than user input. It could be (device disconnected, Tor is disable ...).

I think `interrupted` is good naming, but now reading myself I am thinking that maybe `forcePaused` is more describing.